visual_behavior/dimensionality_reduction/clustering/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .plotting import *
+from .processing import *
```

Comment:
are these imports needed in an init file? they are causing linting to fail

Reply:
These are needed to be able to import functions without specifying `plotting` or `processing`